Update schema_tracking.md (#24146)
DataDog/documentation: content/en/data_streams/schema_tracking.md

</div>
{{% /site-region %}}

> <div class="alert alert-info">Schema tracking is in <b>beta</b> for Java services using Protobuf and Avro. If you're interested in other languages and schemas, let us know here <a href="https://www.datadoghq.com/private-beta/schema-tracking/">here</a>. </div>

Reviewer: ```suggestion
<div class="alert alert-info">Schema tracking is in <b>beta</b> for Java services using Protobuf and Avro. If you're interested in other languages and schemas, complete the form <a href="https://www.datadoghq.com/private-beta/schema-tracking/">here</a>. </div>
```

Author: thanks for the suggestion! I originally opted for "let us know here" since "complete this form" might seem like a lot of work for customers. 

is there any other language we could use?

Reviewer: Thanks for confirming. Would "reach out here" potentially work?

Author: Reach out here sounds great!
